js/analyses/countLinks.js

```diff
@@ -0,0 +1,49 @@
+/** @module analyses/countLinks */
```

Comment:
Same things as above. This is not simply counting links in a text, but doing much more. `getLinkStatistics` is probably better.